core/embed/rust/librust_qstr.h.mako

```diff
@@ -28,6 +28,10 @@ ALTCOIN_PREFIXES = (
     "u2f",
 )
 
+DEBUG_PREFIXES = (
+  "debug",
+)
```

Comment:
Nit: Would be nice to have these prefix lists in one place, not sure where though.

Thinking if we can get rid of the lists entirely and use prefixes based on rust features. Probably too much magic and would not solve `librust_qstr.h` anyway.

Reply:
Good idea, thanks!
Moved prefix lists to `cointool.py` in dabd98f777, which helped to exclude Nostr-related QSTR from `BITCOIN_ONLY` build :)